### ISSUE TITLE
WADark does not change any colors

### ISSUE THREAD

**Issue by algotr8der** [m4heshd/whatsapp-desktop-dark#250]
The WADark CLI and web application runs successfully. However, when WhatsApp reloads none of the selected color themes are applied. Instead, its still white. Everything was working fine until I shutdown WhatsApp Desktop and relaunched today. It had been running for weeks without any problems and I believe the last time I used WADark was maybe 1-2 weeks ago. All was fine then.

WADark Version: v0.4.930
WhatsApp Desktop: Version 2.2023.2
MacOS: 10.15.2 (19C57)

**Comment by timothydillan** [comment]
https://github.com/m4heshd/whatsapp-desktop-dark/issues/242
^ this fix worked for me. Use the Whatsapp Dark theme so that it works.

**Comment by firefinchdev** [comment]
I have fixed the issue. Follow https://github.com/firefinchdev/whatsapp-desktop-dark

**Comment by mechanizzed** [comment]
> I have fixed the issue. Follow https://github.com/firefinchdev/whatsapp-desktop-dark

Thanks!

**Comment by algotr8der** [comment]
Nope not fixed at all. Same problem persists.

**Comment by mechanizzed** [comment]
> Nope not fixed at all. Same problem persists.
> […](#)
> On 6/9/20, Luciano J Guerra ***@***.***> wrote: > I have fixed the issue. Follow > https://github.com/firefinchdev/whatsapp-desktop-dark Thanks! -- You are receiving this because you authored the thread. Reply to this email directly or view it on GitHub: [#250 (comment)](https://github.com/m4heshd/whatsapp-desktop-dark/issues/250#issuecomment-641265642)


You did it?
Open WhatsApp Desktop -> Settings -> Chats -> Theme and select Dark.

**Comment by firefinchdev** [comment]
> Nope not fixed at all. Same problem persists.
> […](#)
> On 6/9/20, Luciano J Guerra ***@***.***> wrote: > I have fixed the issue. Follow > https://github.com/firefinchdev/whatsapp-desktop-dark Thanks! -- You are receiving this because you authored the thread. Reply to this email directly or view it on GitHub: [#250 (comment)](https://github.com/m4heshd/whatsapp-desktop-dark/issues/250#issuecomment-641265642)

Try as said by @mechanizzed

**Comment by spiritualexpander** [comment]
Guys i cant open the WADark file, macOS says "the file is damaged and cant be opened you should move it to the trash" any advice in how to do open it? thanks

**Comment by mechanizzed** [comment]
> Guys i cant open the WADark file, macOS says "the file is damaged and cant be opened you should move it to the trash" any advice in how to do open it? thanks

For macOS you'll need to make WADark executable first, by executing chmod +x WADark from the root of extracted directory.

**Comment by spiritualexpander** [comment]
> > Guys i cant open the WADark file, macOS says "the file is damaged and cant be opened you should move it to the trash" any advice in how to do open it? thanks
> 
> For macOS you'll need to make WADark executable first, by executing chmod +x WADark from the root of extracted directory.

Thanks!

**Comment by Milkeh** [comment]
> > Nope not fixed at all. Same problem persists.
> > […](#)
> > On 6/9/20, Luciano J Guerra _**@**_.***> wrote: > I have fixed the issue. Follow > https://github.com/firefinchdev/whatsapp-desktop-dark Thanks! -- You are receiving this because you authored the thread. Reply to this email directly or view it on GitHub: [#250 (comment)](https://github.com/m4heshd/whatsapp-desktop-dark/issues/250#issuecomment-641265642)
> 
> Try as said by @mechanizzed

This is great. Thanks Firefinch

**Comment by algotr8der** [comment]
Yes I did and I don't even have the following options under Settings:

Chats -> Theme and select Dark.

No where can I find Chats > Theme in any of the menu items within
Whatsapp Desktop.

My Whatsapp Destkop Version 2.2025.7.

My WADark version = 0.4.930.

I can only get it to work by using a very old version of WA Dark.

**Comment by algotr8der** [comment]
I got it to work using  v2.2023.2 and the menu path is:

WhatsApp Desktop > Settings > Theme (select Dark).
